packages/contracts-bedrock/scripts/DeployOwnership.s.sol

```diff
@@ -121,13 +126,38 @@ contract DeployOwnership is Deploy {
         console.log("New LivenessModule deployed at %s", address(addr_));
     }
 
+    /// @notice Deploy a DeputyGuardianModule for use on the Security Council Safe.
+    ///         Note this function does not have the broadcast modifier.
+    function deployDeputyGuardianModule() public returns (address addr_) {
+        Safe councilSafe = Safe(payable(mustGetAddress("SecurityCouncilSafe")));
+        SuperchainConfig superchainConfig = SuperchainConfig(makeAddr("SuperchainConfig"));
+        addr_ = address(
+            new DeputyGuardianModule({
+                _safe: councilSafe,
+                _superchainConfig: superchainConfig,
+                _deputyGuardian: mustGetAddress("FoundationSafe")
+            })
+        );
+
+        save("DeputyGuardianModule", addr_);
+        console.log("New DeputyGuardianModule deployed at %s", addr_);
+    }
+
     /// @notice Deploy a Security Council with LivenessModule and LivenessGuard.
     function deployAndConfigueSecurityCouncilSafe() public returns (address addr_) {
         Safe safe = Safe(payable(deploySafe("SecurityCouncilSafe")));
 
+        vm.startBroadcast();
+        address deputyGuardianModule = deployDeputyGuardianModule();
+        _callViaSafe({
+            _safe: safe,
+            _target: address(safe),
+            _data: abi.encodeCall(ModuleManager.enableModule, (deputyGuardianModule))
```

Comment:
Should we add `deputyGuardianModule` to the `SecurityCouncilConfig` struct, then load the example config to get the address we use here? Currently the `SecurityCouncilConfig` is incomplete